apps/client/src/shared/apis/timerApi.ts

```diff
@@ -48,3 +49,8 @@ export const getBookmarkCards = async (
   );
   return response.data;
 };
+
+export const getAcornCount = async (): Promise<AcornCountResponse> => {
+  const response = await apiRequest.get<AcornCountResponse>('/api/acorn-count');
+  return response.data;
```

Comment:
```suggestion
const { data } = await apiRequest.get<AcornCountResponse>('/api/acorn-count');
return data;
```

data만 return해서 사용할 것이라면 바로 받을 때 구조 분해 할당도 가능해요!

Reply:
이번 API는 테스트 용도로 작성된 것이라, 실제 연동 시에는 응답 구조에 맞춰 구조 분해 할당 등 리팩토링하도록 하겠습니다!! PR에 미리 작성해두지 못해 죄송합니다 🙏